python/sidecar/typing.py

```diff
@@ -176,6 +178,28 @@ class ChatResponse(RefStudioModel):
     choices: list[ChatResponseChoice]
 
 
+class SearchRequest(RefStudioModel):
+    query: str
+    limit: int = 10
+
+
+class S2SearchResult(BaseModel):
```

Comment:
Just noticed that you need to use `RefStudioModel` instead of `BaseModel` for a cleaner output.

Reply:
thank you for finding this. Fixed here - https://github.com/refstudio/refstudio/pull/313/commits/0e758a3fad237872a258a4450bf5eed644b49cb5